Assets/_Game/Scripts/Checkpoint/CheckpointTrigger.cs

```diff
@@ -21,14 +22,19 @@ public class CheckpointTrigger : MonoBehaviour
     private void Awake()
     {
         candleController = GameObject.FindGameObjectWithTag("Candle").GetComponent<CandleController>();
+        activatePopup = transform.GetChild(0).gameObject;
```

Comment:
GetChild is fragile to hierarchy changes, You're already serializing this, why are you manually finding it again?

Reply:
Removed